site/docs/archive.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Archive"
+---
+<!--
+ - Licensed to the Apache Software Foundation (ASF) under one or more
+ - contributor license agreements.  See the NOTICE file distributed with
+ - this work for additional information regarding copyright ownership.
+ - The ASF licenses this file to You under the Apache License, Version 2.0
+ - (the "License"); you may not use this file except in compliance with
+ - the License.  You may obtain a copy of the License at
+ -
+ -   http://www.apache.org/licenses/LICENSE-2.0
+ -
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+ -->
+
+Documentations of Iceberg versions older than 1.4.0 are no longer maintained. Here are the links to web archive.
+
+- [1.3.1](https://web.archive.org/web/20231210001939/https://iceberg.apache.org/docs/1.3.1/)
```

Comment:
I couldn't access this archive

<img width="893" alt="image" src="https://github.com/apache/iceberg/assets/5889404/d4526e2d-72cf-42cb-909b-495bbf6a023d">

Reply:
works now 👍